datastore/cloud-client/requirements-test.txt

```diff
@@ -1,3 +1,4 @@
 backoff==1.10.0
 pytest==5.3.2
+retrying==1.3.3
```

Comment:
retrying is an old library and not maintained. Can you use `backoff` instead?

Reply:
No problem, I've just taken it from another sample.